languages/python/exercises/concept/strings/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+# Instructions
+
+In this exercise you'll be processing log-lines.
+
+Each log line is a string formatted as follows: `"[<LEVEL>]: <MESSAGE>"`.
+
+There are three different log levels:
+
+- `INFO`
+- `WARNING`
+- `ERROR`
+
+You have three tasks, each of which will take a log line and ask you to do something with it.
+
+### 1. Extract a message from a log line
+
+Implement a method to return a log line's message:
+
+```python
+loglines.extract_message("[ERROR]: Invalid operation")
```

Comment:
Personally I think we can lose the module qualification in the name of each function. The only prerequisite for this exercise is `functions`, we will have to have a distinct concept for `modules` and `namespaces`.